contract/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 pub mod primitives;
 
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
-use near_sdk::collections::LookupMap;
+use near_sdk::collections::TreeMap;
```

Comment:
what's the difference between the TreeMap and LookupMap?

Reply:
The lookup map does not know what is inside of it. It just stores elements directly using the formula:
`map_tag` + `<element key>`. TreeMap builds a tree of elements. In our case, LookupMap is more efficient, but probably it's not that important, since the max size of the map is 8.

Reply:
but what specifically do we get out of using `TreeMap` over `LookupMap`? Doesn't seem clear to me what the benefit here is

Reply:
It is not possible to know the size of the `LookupMap`.